### PR TITLE
Fixed Syntax

### PR DESCRIPTION
**Pull request by Bonjur**

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks very much. However this fix was incorrect - having the bracket after the  comma implies that this is a valid format: 
```
pipeThrough(<em>transformStream</em>, );
```
rather than:
```
pipeThrough(<em>transformStream</em>);
pipeThrough(<em>transformStream</em>, <em>options</em>);
```
We're moving to expand out the definitions multi-line to remove this confusion for users. I have made the change as a patch. @Elchi3 can you please review/merge? (also, shouldn't all of the em's be var's?)

File: files/en-us/web/api/readablestream/pipethrough/index.html [hunk 1]
```diff
@@ -21,8 +21,9 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre
-  class="brush: js notranslate">var <em>transformedStream</em> = <em>readableStream</em>.pipeThrough(<em>transformStream</em>[, <em>options</em>]);</pre>
+<pre class="brush: js">pipeThrough(<em>transformStream</em>);
+pipeThrough(<em>transformStream</em>, <em>options</em>);
+</pre>
```
Comment (Elchi3) [inline files/en-us/web/api/readablestream/pipethrough/index.html:26]:
```suggestion
<pre class="brush: js">pipeThrough(<var>transformStream</var>);
pipeThrough(<var>transformStream</var>, <var>options</var>);
</pre>
```

**Review by Elchi3 — COMMENTED** [review]
Another proof that the BNF syntax is really confusing. 😄 

Changing to multiple lines is great here, superb Hamish! I also agree about `<var>` being the better markup here.

**Review by hamishwillee — APPROVED** [review]
Thanks @Bonjur ! Final version updated in line with "latest and greatest MDN approach of using multi-lines" :-)

PS thanks also for review @Elchi3

**Pull request merged**